terraform/main.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4"
```

Comment:
Please bump the version to 5. It's been release a few month back. and run `make lock-provider` to update terraform lock